test/get.coffee

```diff
@@ -304,3 +304,28 @@ describe 'quest', ->
         quest options, (err, resp, body) ->
           assert.ifError err
           done()
+
+      it 'is thenable on success', (done) ->
+        @timeout 20000
+        options =
+          uri: "#{protocol}://httpbin.org/get"
+        quest options
+          .then (resp) ->
```

Comment:
That is correct - for more info see [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise). Then() takes 2 functions - a success callback, and a failure callback.